pkg/certs/util.go

```diff
@@ -68,3 +71,51 @@ func CertificateBundleToString(bundle []*x509.Certificate) string {
 	}
 	return strings.Join(output, "\n")
 }
+
+func ValidatePrivateKeyFormat(pemKey []byte) []error {
```

Comment:
Wasn't there supposed to be a godoc here as well?

Reply:
Added